### ISSUE TITLE
Train loss

### ISSUE THREAD

**Issue by vanduong0504** [zylo117/Yet-Another-EfficientDet-Pytorch#567]
I trained a brand logo detection (gucci, adidas,...) and followed your shape tutorial (using efficientdet-d0.pth). But the loss don't change much from epoch 100 to 200 then it stopped with early stoping. My AP score is  when testing with 0.002. 
My coco format don't have segmentation like you, can't it be ok?  [My data google drive here](https://drive.google.com/file/d/1-Sm6E_lwj1esqkDTU4ozaf_F3nsuD5Pb/view?usp=sharing). Can you check it for me?

Here my training code:

    ! python train.py -c 0 -p logo --head_only True --lr 1e-3 --batch_size 32 --load_weights weights/efficientdet-d0.pth  
    --num_epochs 500 --es_patience 20  --save_interval 100 
And my .yml file:

    #project_name: logo  # also the folder name of the dataset that under data_path folder
    train_set: train
    val_set: val
    num_gpus: 1

    # mean and std in RGB order, actually this part should remain unchanged as long as your dataset is similar to coco.
    mean: [0.485, 0.456, 0.406]
    std: [0.229, 0.224, 0.225]

    # this anchor is adapted to the dataset
    anchors_scales: '[2 ** 0, 2 ** (1.0 / 3.0), 2 ** (2.0 / 3.0)]'
    anchors_ratios: '[(1.0, 1.0), (1.4, 0.7), (0.7, 1.4)]'

    obj_list: ['adidas0', 'chanel','gucci','hh','lacoste','mk','nike','prada','puma','supreme']
What do I need to improve performance? Thank you

**Comment by zylo117** [comment]
cool, I'll try training this one.

**Comment by vanduong0504** [comment]
Thank you, may I have your mail for discussing more?

**Comment by zylo117** [comment]
Hi, I've just trained it to mAP 0.736(iou 0.5) in 15 minutes in colab.
Do you mind me making it into a public tutorial?

**Comment by vanduong0504** [comment]
Sure, if you want. Can you share me?

**Comment by zylo117** [comment]
of course, I'll upload it soon.

**Comment by vanduong0504** [comment]
Please share it my mail: **vanduong0504@gmail.com** in today because I need to check it as soon as possible. You can upload it to public tutorial any time you want. Sorry for the inconvenience. I owe you big time.

**Comment by zylo117** [comment]
it's uploaded. thanks for your contributions!
https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/2b529a5d57626dbee47b727836c6a7c2687e0ee3/tutorial/train_logo.ipynb

**Comment by zylo117** [comment]
I also published the best weights I got. https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/releases/download/custom_datasets/logo_efficientdet-d0.pth

**Comment by vanduong0504** [comment]
Thanks
